qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -205,67 +205,73 @@ def spin2(electrons, orbitals, mapping="jordan_wigner", wires=None):
     Coulomb interaction between the electrons.
 
-    If an `active space <https://en.wikipedia.org/wiki/Complete_active_space>`_
-    (see :func:`~.active_space`) is defined, the observable is expanded over the truncated
-    basis of active orbitals. The contribution of core orbitals can be passed to the
-    function using the keyword argument ``init_term``.
+    - The observable is built by adding the operators
+      :math:`\sum_{\alpha, \beta} t_{\alpha\beta}^{(i)}
+      \hat{c}_\alpha^\dagger \hat{c}_\beta` and
+      :math:`\frac{1}{2} \sum_{\alpha, \beta, \gamma, \delta}
+      v_{\alpha\beta\gamma\delta}^{(i)}
+      \hat{c}_\alpha^\dagger \hat{c}_\beta^\dagger \hat{c}_\gamma \hat{c}_\delta`.
+
+    - All second-quantized one-particle and two-particle operators contributing to the
+      many-body observable must be represented using the `FermionOperator
+      <https://github.com/quantumlib/OpenFermion/blob/master/docs/
+      tutorials/intro_to_openfermion.ipynb>`_ data structure as implemented in OpenFermion.
 
-    The function uses tools of `OpenFermion <https://github.com/quantumlib/OpenFermion>`_
-    to build the second-quantized operator and map it to basis of Pauli matrices via the
-    Jordan-Wigner or Bravyi-Kitaev transformation. Finally, the qubit operator is
-    converted to a a PennyLane observable by the function :func:`~.convert_observable`.
+    - The function uses tools of `OpenFermion <https://github.com/quantumlib/OpenFermion>`_
+      to map the resulting fermionic Hamiltonian to the basis of Pauli matrices via the
+      Jordan-Wigner or Bravyi-Kitaev transformation. Finally, the qubit operator is converted
+      to a PennyLane observable by the function :func:`~.convert_observable`.
 
     Args:
-        matrix_elements (list(array[float])): list containing 2D numpy arrays with the matrix
-            elements of the operators :math:`\hat{t}` and :math:`\hat{v}`.
-            For single-particle operators the :math:`i`-th array in the list will have shape
-            ``(matrix_elements[i].shape[0], 3)`` with each row containing the indices
-            :math:`\alpha`, :math:`\beta` and the matrix element
-            :math:`\langle \alpha \vert \hat{t}^{(i)}\vert \beta \rangle`.
-            For two-particle operators the :math:`j`-th array in the list
-            will have shape ``(matrix_elements[j].shape[0], 5)`` with each row containing
-            the indices :math:`\alpha`, :math:`\beta`, :math:`\gamma`, :math:`\delta` and
-            the matrix element
-            :math:`\langle \alpha, \beta \vert \hat{v}^{(j)}\vert \gamma, \delta \rangle`.
-        init_term (float): the contribution of core orbitals, if any, or other quantity
-            required to initialize the many-body observable.
-        mapping (str): specifies the fermion-to-qubit mapping. Input values can
+        fermion_ops (list[FermionOperator]): list containing the FermionOperator data structures
```

Comment:
Is there a more illustrative way to define the format instead of using list[FermionOperator]? When I look at the example below, it seems that we have [FermionOperator + FermionOperator, FermionOperator + FermionOperator]. Not sure what is the best way though.

Reply:
Does `FermionOperator + FermionOperator` return a new `FermionOperator`? If so, this should be the correct way of writing it

Reply:
Yes.